test/fs-writable.js

```diff
@@ -45,6 +45,15 @@ metatests.test('fs-writable / open', test => {
   test.on('done', () => fs.unlinkSync(TEST_FILENAME));
 });
 
+metatests.test('fs-writable / open with error', test => {
+  const TEST_FILENAME = __dirname;
+
+  test.plan(1);
+
+  const stream = new WritableFileStream(TEST_FILENAME);
+  stream.on('error', () => test.pass('open'));
```

Comment:
I'd prefer to explicitly check error that was emitted. (to avoid this test passing when some other error is emitted)